docs/product-getting-started-guide/src/main/asciidoc/test-speed-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='testing_speed_limit-proc']
+= Testing the speed limit scenario
+
+The following chapter instructs you how to test the speed limit scenario using the data that you specified when you created the speed violation guided decision table.
+
+[float]
+== Prerequisites
+
+* Created the Driver department traffic violations project
```

Comment:
Monospaced formatting, etc. See my comment in Testing the driver license section.

Reply:
Done